### PR TITLE
js integration test orbs + other orbs cleanup

### PR DESCRIPTION
**Pull request by magrytos**
Refactor, cleanup and new orbs for FE. This PR covers all use cases for FE pipelines, meaning every FE config can be composed only with orbs. This will be a major release.
- testing jobs using parallelism for `cypress` and `eslint`, refactoring `jest` job
- all logic related to `tmp-test-list` file was moved into orbs to simplify ci related scripts in `package.json`
- force scripts to be named `ci:*`:
  - `ci:build`
  - `ci:build-storybook`
  - `ci:cypress:beta`
  - `ci:cypress`
  - `ci:lint`

Some proofs that it works:
[listing-form](https://app.circleci.com/pipelines/github/ricardo-ch/listing-form-spa?branch=orbs-only)
[marketplace-frontend-app](https://app.circleci.com/pipelines/github/ricardo-ch/marketplace-frontend-app?branch=ci-improvements)

### CODE REVIEW

File: src/commands/js_docker_prebuild_steps.yml
Comment (magrytos) [inline src/commands/js_docker_prebuild_steps.yml:1]: .env.sentry is not used anymore

File: src/jobs/js_jest_test_runner_job.yml
Comment (magrytos) [inline src/jobs/js_jest_test_runner_job.yml:1]: This file was modified and renamed to js_jest_runner_job.yml

File: src/jobs/js_yarn_runner_job.yml
Comment (magrytos) [inline src/jobs/js_yarn_runner_job.yml:1]: only file structure changes here

**Review by mbriziarelli — CHANGES_REQUESTED** [review]
LGTM. except the remaining js_jest_runner_job.yml

**Review by mbriziarelli — APPROVED** [review]

**Review by mbriziarelli — APPROVED** [review]

**Pull request merged**